extensions/chromium/claude-api-handler.js

```
@@ -0,0 +1,60 @@
+chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
+  if (request.action === "callClaudeAPI") {
+    console.log("Background script received request to call Claude API");
+    
+    // Extract API details from the request
+    const { prompt, model, apiKey } = request.data;
+
+    console.log("Prompt:", prompt);
+    console.log("Model:", model);
+    
+    // Hard-coded API key that we know works with the curl command
+    const hardcodedApiKey = "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";
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)